src/rundeck/values.yaml

```diff
@@ -0,0 +1,115 @@
+# Default values for rundeck.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: deepakr11/rundeck
+  tag: 0.2.0
+  pullPolicy: Always
+
+service:
+  type: ClusterIP
+  http:
+    port: 4440
+  https:
+    port: 4443
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - rundeck.default.svc.cluster.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  #limits:
+  #  cpu: 1
+  #  memory: 1024Mi
+  #requests:
+  #  cpu: 100
+  #  memory: 512Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+#####################################################################
+# Application Configurations
+
+#######################################
+# Pod Spec Overrides
+
+#appname: rundeck
+
+env:
+  SHARED_STORAGE: /rundeck
```

Comment:
I don't see sstarcher's comments taken care of
<img width="484" alt="image" src="https://user-images.githubusercontent.com/1733071/43166884-b0e1345a-8f4c-11e8-821d-1efb47b4c06a.png">

Reply:
This particular scenario, we cannot do as we are defining env variables and that should be defined as flat structure. Other places are covered.